tests/unit/plugins/modules/identity/test_bitwarden_info.py

```diff
@@ -0,0 +1,510 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2022, Jonathan Lung (@lungj) <lungj@heresjono.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible_collections.community.general.tests.unit.compat import unittest
+from ansible_collections.community.general.tests.unit.compat.mock import patch
+from ansible_collections.community.general.tests.unit.plugins.modules.utils import (
+    AnsibleExitJson, AnsibleFailJson, set_module_args
+)
+
+from ansible_collections.community.general.plugins.modules.identity import bitwarden_info
+
+from ansible_collections.community.general.tests.unit.plugins.modules.identity.test_bitwarden import (
+    ORGANIZATIONS, FOLDERS, ITEMS, MOCK_RESPONSES, MockClient, client, mock
```

Comment:
For the sake of code organization, I suggest refactoring those elements out of the `test_bitwarden.py` file and importing them in both testcase files.

Reply:
Do you have any suggested package/module names? My first reaction would be to group things in a `bitwarden` package and put in a `conftest`, but I know the hashi_vault folks chose that name to avoid any (accidental) connection to Hashicorp. I also tried (unsuccessfully) to put everything in one module to avoid this coming up in the first place.